compiler/rustc_target/src/spec/targets/aarch64_apple_xros.rs

```diff
@@ -0,0 +1,21 @@
+use crate::spec::base::apple::{opts, xros_llvm_target, Arch};
+use crate::spec::{FramePointer, SanitizerSet, Target, TargetOptions};
+
+pub fn target() -> Target {
+    let arch = Arch::Arm64;
+    let mut base = opts("xros", arch);
+    base.supported_sanitizers = SanitizerSet::ADDRESS | SanitizerSet::THREAD;
+
+    Target {
+        llvm_target: xros_llvm_target(arch).into(),
+        pointer_width: 64,
+        data_layout: "e-m:o-i64:64-i128:128-n32:64-S128".into(),
+        arch: arch.target_arch(),
+        options: TargetOptions {
+            features: "+neon,+fp-armv8,+apple-a7".into(),
```

Comment:
I think this can be much higher, possibly even `apple-a15` or `apple-m2`?

Same for the simulator target.

Reply:
Very good point. Do you know how I would find out what is acceptable? Just poking around in LLVM?

Reply:
See https://github.com/llvm/llvm-project/blob/fde344aef20bc4280f01294ac6e14a5c2db2d572/clang/lib/Driver/ToolChains/Arch/AArch64.cpp#L50-L60

Reply:
I am unsure why, but in use `rustc` reports that `apple-m1` is not a valid feature for this target. I see nothing obvious in the codebase where we whitelist LLVM architectures.

Reverting to the latest available in `rustc`, `apple-a16`